modules/civicrm_rules/civicrm_rules.contact-eval.inc

```diff
@@ -144,3 +144,24 @@ function civicrm_rules_options_list() {
   $settings['groups'] = _civicrm_get_groups();
   return $settings;
 }
+
+/**
+ * Options list callback for listing of CiviCRM Tags
+ */
+function civicrm_rules_tags_list() {
+  if(!civicrm_initialize()) {
```

Comment:
@agileware-fj need a space after the `if`

Reply:
added